### ISSUE TITLE
retire RepresentationType

### ISSUE THREAD

**Issue by I-Sokolov** [buildingSMART/IFC_UML#57]
Description of the proposal: remove IfcProduct.RepresentationType attribute
What do we win: simplify and consistent schema, remove redundant data.
All the information may be obtained from classes of IfcRepresentationItems

**Comment by berlotti** [comment]
Great catch!
Seems redundant information indeed.

**Comment by TLiebich** [comment]
Hi @I-Sokolov - just to confirm - you mean IfcRepresentation.RepresentationType, not IfcProduct.RepresentationType?
https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/link/ifcrepresentation.htm

we should discuss, since I know that some software tries to rely on it, although it is difficult to trust (as you need to check the SET of IfcRepresentation.Items anyhow).

**Comment by I-Sokolov** [comment]
Yes IfcRepresentation.RepresentationType  (there is no IfcProduct.RepresentationType attribute)
I know products relay on this... I saw crazy code in some application that tries to marriage RepresentationType and content, esecially when they conflict:)
